.github/workflows/continous_integration.yaml

```
@@ -74,9 +74,4 @@ jobs:
           pytest -v --cov=physical_validation physical_validation/tests/
 
       - name: Code coverage
-        run: |
-
-          . devtools/scripts/initialize_conda.sh
-          conda activate test
-
-          codecov
+        uses: codecov/codecov-action@v1
```

```suggestion
        uses: codecov/codecov-action@v1
        with:
          file: ./coverage.xml
```
Same as the above comment - I've found in the past I needed to explicitly give codecov the expected file path for it to upload correctly.

---

Good call on explicitly telling pytest to write an xml report file. I though pytest was writing it in an obscure location, but it turns out it wasn't writing it at all!

---

Added those changes and now the this commits report popped up on codecov. (see the codecov-commenter below)